src/main/java/org/pac4j/vertx/verticle/DemoServerVerticle.java

```diff
@@ -121,7 +121,13 @@ public void start() {
         // Requires authentication endpoint without specific authenticator attached
         addProtectedEndpointWithoutAuthorizer("/protected/index.html", "", router);
 
-        router.get("/index.html").handler(DemoHandlers.indexHandler(config));
+        // Direct basic auth authentication (web services)
+        addProtectedEndpointWithoutAuthorizer("/dba/index.html", "DirectBasicAuthClient", router);
```

Comment:
**/rest-jwt/index.html link**:

- If I click on the link, I get a "unauthorized" error: OK.
- If I had "?token=" in the url, I get an internal error: OK, this is the expected behaviour, but the error page is not the right one (I expected: `error500.html`).
- If I provide a good JWT, it works: the profile is properly displayed on the specific page. Though, when I come back to the index page, the profile is still displayed meaning it has been saved in the session while it should have only been available in the request (by default) for this kind of call.